README.md

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
You should have separate repositories for your server&logger, your file-writer app and your api server.